### PR TITLE
Console error :skull: so close

### PR DESCRIPTION
**Pull request by eilinora**
added bool to 3 more components

### CODE REVIEW

File: src/forms/CalendarComponent.jsx [hunk 1]
```diff
@@ -114,20 +108,11 @@ CalendarComponent.propTypes = {
...
 	datepickerOptions: PropTypes.object,
-	error: PropTypes.oneOfType([
-		PropTypes.string,
-		PropTypes.element
-	]),
-	helperText: PropTypes.oneOfType([
-		PropTypes.string,
-		PropTypes.element
-	]),
+	error: PropTypes.oneOfType([PropTypes.string, PropTypes.element, PropTypes.bool]),
```
Comment (eilinora) [inline src/forms/CalendarComponent.jsx]: only real change

File: src/forms/ErrorList.jsx [hunk 2]
```diff
@@ -12,29 +12,22 @@ const ERROR_CLASSNAME = 'text--error text--small';
...
-			{error && <li className={cx(ERROR_CLASSNAME, 'text--small')}>{error}</li> }
+		<ul aria-live="assertive" {...other}>
+			{error && <li className={cx(ERROR_CLASSNAME, 'text--small')}>{error}</li>}
 		</ul>
 	);
 };
 ErrorList.propTypes = {
 	errorId: PropTypes.string,
-	error: PropTypes.string,
+	error: PropTypes.oneOfType([PropTypes.string, PropTypes.element, PropTypes.bool]),
```
Comment (eilinora) [inline src/forms/ErrorList.jsx]: only real change

File: src/forms/TimeInput.jsx [hunk 3]
```diff
@@ -402,7 +402,7 @@ TimeInput.defaultProps = {
 
 TimeInput.propTypes = {
 	name: PropTypes.string.isRequired,
-	error: PropTypes.string,
+	error: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```
Comment (eilinora) [inline src/forms/TimeInput.jsx]: only real change

File: src/forms/CalendarComponent.jsx (hunk 1 above)
Comment (mperrotti) [reply src/forms/CalendarComponent.jsx]: What's the use case for passing a boolean to `error`? I thought we used this prop for rendering an error message

Comment (eilinora) [reply src/forms/CalendarComponent.jsx]: The schedule form is passing false when there is no error to show

**Pull request merged**